.env

```diff
@@ -0,0 +1,2 @@
+# DON'T COMMIT THIS FILE
```

Comment:
👀

Reply:
hahahaha after my commit :)
it's used during tests.